### PR TITLE
add footer

### PR DESCRIPTION
**Pull request by slawick12**

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -23,18 +23,21 @@
               "src/assets"
             ],
             "styles": [
-              "src/styles.css"
+              "src/styles.css",
+              "../node_modules/bootstrap/dist/css/bootstrap.min.css"
```
Comment (Resserr) [inline angular.json]: we dont need that. Bootstrap is imported to styles.css

File: angular.json [hunk 2]
```diff
@@ -23,18 +23,21 @@
...
           "configurations": {
             "production": {
-              "fileReplacements": [
-                {
-                  "replace": "src/environments/environment.ts",
-                  "with": "src/environments/environment.prod.ts"
-                }
-              ],
+              "fileReplacements": [{
+                "replace": "src/environments/environment.ts",
```
Comment (Resserr) [inline angular.json]: unnecessary changes

File: package.json [hunk 3]
```diff
@@ -20,10 +20,13 @@
     "@angular/platform-browser": "^6.0.3",
     "@angular/platform-browser-dynamic": "^6.0.3",
     "@angular/router": "^6.0.3",
-    "bootstrap": "^4.3.1",
+    "@ng-bootstrap/ng-bootstrap": "^4.1.2",
```
Comment (Resserr) [inline package.json]: why did you add ng-boostsrap?

File: package.json [hunk 4]
```diff
@@ -20,10 +20,13 @@
...
+    "bootstrap": "^4.0.0",
```
Comment (Resserr) [inline package.json]: that was already added earlier

File: package.json [hunk 5]
```diff
@@ -20,10 +20,13 @@
     "@angular/platform-browser": "^6.0.3",
     "@angular/platform-browser-dynamic": "^6.0.3",
     "@angular/router": "^6.0.3",
-    "bootstrap": "^4.3.1",
```
Comment (Resserr) [inline package.json]: why did you delete it?

**Comment by slawick12** [comment]
I was trying to make a Burger button


пн, 6 мая 2019 г. в 14:11, Resserr <notifications@github.com>:

**Comment by Resserr** [comment]
потрібно повиправляти всі помилки і розвязати також конфлікт який виник в одному з компонентів
